docs/how-to-run.adoc

```diff
@@ -134,17 +134,31 @@ $ pfexec ./target/release/omicron-package install
 If you've done this already, and you wish to recover, run from the root of this repository `pfexec chown -R $USER:$(id -ng $USER) target ${CARGO_HOME:-~/.cargo}`.
 ====
 
-This service installs a bootstrap service, which itself loads other
-requested services. The bootstrap service is currently the only
-service which is "persistent" across reboots - although it will
-initialize other service as part of its setup sequence anyway.
+This service installs a bootstrap service called
+`svc:/system/illumos/sled-agent:default`, which itself loads other requested
+services. The bootstrap service is currently the only service which is
+"persistent" across reboots - although it will initialize other services as part
+of its setup sequence anyway.
+
+The first time the bootstrap service runs, it will take a while to initialize
+the Omicron zones:
 
 [source,text]
 ----
 # List all services:
 $ svcs
+
+# View logs for sled-agent:
+$ tail -f /var/svc/log/system-illumos-sled-agent:default.log
```

Comment:
I would instead make this:

```
$ tail -F $(svcs -L sled-agent)
```

First, the `tail -F` flag should handle truncation/rotation of the log, which can happen from time to time:

```
       -F
              Follow named file. Operates as with -f, except that if the file
              is moved (e.g. if a watched log file is rotated) tail will close
              the original file and begin reading records from the start of
              the file with the specified name if and when that file is
              recreated.
```

The `-L` argument to [svcs(1)](https://illumos.org/man/1/svcs) emits the path to the log file for a service instance:

```
       -L
                           Display the log file of the selected services and
                           service instances, one per-line.
```

Reply:
Thanks, I keep forgetting about `tail -F`, it's always been super handy.